server/package.json

```diff
@@ -58,7 +58,7 @@
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
     "cookie-parser": "^1.4.6",
-    "exiftool-vendored": "^19.0.0",
+    "exiftool-vendored": "^22.0.0",
```

Comment:
I assume this is fine? I didn't read the release notes though.

Reply:
We need a newer version as older one cannot extract projectiontype